src/js/env/node/index.js

```diff
@@ -36,10 +36,15 @@ export const manifest = (m: $T.Manifest) => {
 };
 
 export const dispose = () => {
-    // iframe.dispose();
-    // if (_popupManager) {
-    //     _popupManager.close();
-    // }
+    eventEmitter.removeAllListeners();
+    if (_core) {
+        _core.dispose();
+    }
+    // TODO: fix top level values during refactor
```

Comment:
What this TODO means, please? What do you mean by those "top level values"?

Reply:
i meant variables :)

_core is a top level variable in this module and this is why // $FlowIssue was required

it needs to be refactored properly, but it is out of scope for this PR